html-forms/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title> HTML Forms </title>
+</head>
+
+<body>
+
+<h1>Coding Form!</h1>
+
+<form action = 'index.html' method = 'post'></form>
+  <label for='mail'>Email</label>
+  <div>
+    <input type='email' id='mail' name='user_email'>
+  </div>
+    <label for='name'>Why is coding cool?</label>
+
+  <div>
+    <select id="simple" name="simple">
+      <option selected>Select an option</option>
+      <option>Because it's interesting</option>
+      <option>Because it's fun</option>
+      <option>Because it's powerful</option>
+      </select>
+  </div>
+
+  <div>
+    <label for='text'>Favorite Language (choose one)</label>
+  </div>
+    <input type="radio" name="fav" id="fav_1" />
+      <label for="fav_1">HTML</label>
+    <input type="radio" name="fav" id="fav_2" />
+      <label for="fav_2">CSS</label>
+    <input type="radio" name="fav" id="fav_3" />
+      <label for="fav_3">Javascript</label>
```

Comment:
`id` values should never have an underscore. It should either be camelCased or kebab-cased